### PR TITLE
Update unit tests to remove some downloading

### PR DESCRIPTION
**Pull request by jlmaurer**
This PR focuses on streamlining unit tests to avoid downloading when possible

## Description

- removes downloads from test_datelist

## Motivation and Context

- repeated download failures make it hard to test

## How Has This Been Tested?

- updated tests pass locally

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ x] unit test update

## Checklist:

- [ ] I have added an explanation of what your changes do and why you'd like us to include them.
- [ ] I have written new tests for your core changes, as applicable.
- [x] I have successfully ran tests with your changes locally.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by jlmaurer** [comment]
@bbuzz31 can you take a quick look at the error message on this? This PR is passing locally and I ran the test independently and completes just fine.

File: test/scenario_5/temp.yaml
Comment (bbuzz31) [inline test/scenario_5/temp.yaml:1]: can / should this yaml file be created on the fly like the other ones?

Comment (bbuzz31) [reply test/scenario_5/temp.yaml:1]: Also, can/should it point to the global testing WM_DIR (RAiDER/test/weather_files)?

File: test/test_GUNW.py
Comment (bbuzz31) [inline test/test_GUNW.py:1]: This test will get skipped due to `@pytest.mark.long?` yes?
I think we'd prefer that not to happen, as the GUNW workflow will be cranking lots of products

**Review by bbuzz31 — APPROVED** [review]

**Pull request merged**